### PR TITLE
initial translate to Thai language

### PR DESCRIPTION
**Pull request by merxer**

### CODE REVIEW

File: th/lessons/advanced/concurrency.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
+---
+layout: page
+title: Concurrency
+category: advanced
+order: 4
+lang: en
```
Comment (doomspork) [inline th/lessons/advanced/concurrency.md]: We need to remove all of the `en` lessons from this PR, unfortunately they will screw up the generated content.

**Comment by doomspork** [comment]
Thank you @merxer!  Could you please remove the English lessons and rebase your branch?  Thanks

**Comment by merxer** [comment]
Completed to change default English to Thai
Thanks

**Comment by doomspork** [comment]
@merxer it looks like this needs to be rebased before it can merge

**Comment by doomspork** [comment]
Hi @merxer, any chance you can rebase so we can merge your changes in and announced the start of the Thai translations?

**Comment by merxer** [comment]
rebase completed,

**Comment by merxer** [comment]
Hi Sean,

Sorry for late reply, completed to rebase

:)

Thanks

**Comment by doomspork** [comment]
@merxer could we open a new PR from your master branch?  The rebase seemed to add _a lot_ more commits to your PR

**Comment by merxer** [comment]
Dear Sean,

 NO, I need to start new translate and then open new PR to you

BRs,
Thawatchai S.

**Pull request closed without merging**